examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Maybe we can add sentence to say that we are also supporting .json schema format (like this <a href="https://github.com/NVIDIA-Merlin/models/blob/main/merlin/models/data/ecommerce/small/schema.json" target="_blank">one</a>) if the user wants to use another feature engineering library?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>